resolution-aware-images.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: page
+title: Resolution Aware Images
+permalink: /resolution-aware-images/
+---
+
+Flutter can load resolution-appropriate images for the current device
+pixel ratio.
+
+*Tip*: Remember to declare all of your resources inside `flutter.yaml`,
+as shown in the [Tutorial](/tutorial/).
```

Comment:
I forgot to mention, but we should explain somewhere in here, that Flutter's build phase now automatically picks up any asset variants in subfolders that have the same name as a main asset.  So if you declare my_icon.png in the example below, the build phase will automatically include 2.0x/my_icon.png and 3.0x/my_icon.png in the asset bundle.